mousestyles/path_diversity/smooth_noise.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function, absolute_import, division
+
+import pandas as pd
+import numpy as np
+from path_features import compute_angles
+from detect_noise import detect_noise
+
```

Comment:
Should be 
`from mousestyles.path_diversity import compute_angles`
and 
`from mousestyles.path_diversity import detect_noise`
as @jJasonWang and @sicihuang suggested?